qdrant-landing/content/documentation/cloud/configure-cluster.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Configure Clusters
+weight: 55
+---
+
+# Configure Qdrant Cloud Clusters
+
+Qdrant Cloud offers several advanced configuration options to optimizer your clusters to your specific needs. You can access these options from the Cluster details page in the Qdrant Cloud console.
+
+## Collection Defaults
+
+You can set default values for the configuration of new collections in your cluster. These defaults will be used when creating a new collection, unless you override them in the collection creation request.
+
+You can configure the default *Replication Factor*, the default *Write Consistency Factor* and if vectors should be stored on disk only, instead of being cached in RAM.
```

Comment:
I think we conventionally use the Oxford comma here so there should be a comma before 'and'

Also I think the final clause could be instead but this is a stylistic opinion

', and if vectors should be cached in RAM or only stored on disk.'